Camera/AmetekS711EuresysCoaxlinkQsfp+/camera_phantom_s711.hh

```diff
@@ -249,6 +272,14 @@ class EHoloGrabber
     /*! \brief An EGrabbers instance composed of the two EHoloSubGrabber grabbers.  */
     EGrabbers<EHoloSubGrabber> grabbers_;
```

Comment:
You can delete this attribute since it's only used in the ctor (move it to a local variable in the ctor).